LWM2M_Portfolio-v1_0_1.xml

```diff
@@ -15,17 +15,17 @@ NORMATIVE INFORMATION
 
   Information about this file can be found in the latest revision of
 
-    OMA-TS-LWM2M_PortfolioObj-V1_0
+    OMA-TS-LWM2M_PortfolioObj-V1_0_1
 
-  This is available at http://www.openmobilealliance.org/
+  This is available at https://www.omaspecworks.org/
```

Comment:
@jmudge Why do we have omaspecworks.org for Technical documents? We decided that we won't  change that location.

Reply:
@jpradocueva the applicable template is https://github.com/OpenMobileAlliance/OMA_Document_Templates/blob/master/SUP/n.xml and this includes https://www.omaspecworks.org/ . I don't know when or whom initiated this change but it was over a year ago. If it is required to change the URL in these XML files then this template would also need to be changed.

Furthermore please note that http://www.openmobilealliance.org/ automatically redirects to https://www.omaspecworks.org/ . People have contacted the Helpdesk complaining that are unwilling to use the auto diverted URL https://www.omaspecworks.org/ as they feared that the site http://www.openmobilealliance.org/ has been hijacked to redirect to an illegitimate site.

Please confirm if you wish the URL in these XML files and also in the template to be changed.